### PR TITLE
Feature controller conflict resolved: Update jobPost

### PR DESCRIPTION
**Pull request by Nejo12**

### CODE REVIEW

**Review by dilippoudel — APPROVED** [review]

File: test/controllers/employer.test.ts [hunk 1]
```diff
@@ -110,10 +109,35 @@ describe('user controller', () => {
...
+      .post('/employer/jobs/google')
+      .send(jobPost)
+
+    const jobPostId = response.body.savedJobPost.id
+
+    const update = {
+      title: 'Updated job title',
+      jobDescription: 'Updated Job Description',
+      seniority: 'Junior',
+    }
```
Comment (Wyfy0107) [inline test/controllers/employer.test.ts]: Can you improve this by update only what is included in the update form

**Comment by dilippoudel** [comment]
Nice @Nejo12  👍

**Review by Wyfy0107 — CHANGES_REQUESTED** [review]
👍 Good job. Please update the controller and I can merge it

File: test/controllers/employer.test.ts (hunk 1 above)
Comment (Nejo12) [reply test/controllers/employer.test.ts]: Improvement made on the update form. As discussed, await Skill table creation, thereby omitting it for now.

**Review by Nejo12 — COMMENTED** [review]
updateJobPost rectified.

**Review by khumair — APPROVED** [review]

**Pull request closed without merging**